src/applications/using-buildpacks.md

```diff
@@ -2,6 +2,21 @@
 
 Deis supports deploying applications via [Heroku Buildpacks][]. Buildpacks are useful if you're interested in following Heroku's best practices for building applications or if you are deploying an application that already runs on Heroku.
 
+## Add SSH Key
+ 
+For **Buildpack** based application deploys via `git push`, Deis Workflow identifies users via SSH keys. SSH keys are pushed to the platform and must be unique to each user.
+
+How to generate the SSH key check it [here](../users/ssh-keys.md/#generate-an-ssh-key).
```

Comment:
Can this say `See [this document](../users/ssh-keys.md/#generate-an-ssh-key) for instructions on how to generate an SSH key`?